pkg/runtime/wfengine/backend.go

```diff
@@ -108,8 +109,12 @@ func (be *actorBackend) CreateOrchestrationInstance(ctx context.Context, e *back
 
 	resp, err := be.actors.Call(ctx, req)
 	if err != nil {
+		// Failed request to create workflow, record metrics.
+		diag.DefaultWorkflowMonitoring.WorkflowOperationsFailed(ctx, "Create Workflow")
```

Comment:
Similarly define another constant in workflow_metrics and use that. And maybe `create_workflow` instead of `Create Workflow`?

Reply:
fixed